scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+python-dateutil
+pytz
+requests
```

Comment:
Ideally these dependencies should be pinned, but I think it would make sense to wait to see if we want to put more effort into containerizing the test data ETL before doing that. My Python environment is substantially different such that I'm not confident the versions of these packages that I installed will generalize to all machines.

Reply:
Hmm yeah, it might actually be best to hold off on this file altogether for the moment. If we add containerization it won't be necessary, and if this was what you needed to get things going for your own local machine, then it's likely to be incomplete because you probably had some stuff installed already. What might be good is to include these in the list of requirements you added to the README; then we can provide better context and explain that the listed requirements are not necessarily complete.

Reply:
I like that solution -- much more clearly communicates the level of confidence I have in the completeness of the dependencies.